assignments/a6.md

```diff
@@ -51,7 +56,33 @@ It is suggested to use a folder and multiple files, just like this CSC491 repo,
+| Test coverage | Tests should cover at least 65% of your application, upload to the DCSIL Team App, and upload to Code Climate. | 20.0 |
+| Code Quality | All repositories have at least a "B" Grade on Code Climate. | 10.0 |
+| Operational Excellence | CI, Exceptions, Logging, and all other services listed in the requirements & `service.yml` example are all setup. Bootstrap script works. Service.yml is uploaded with no errors in the DCSIL team app. | 10.0 |
+| Dynamic | The majority of the content is functional and dynamic. The apps do not rely on hard-coded static data. | 10.0 |
+| Use Cases | All main use cases/flows have been delivered and are functional. | 15.0 |
+| Video | A short video with a short write up (some bullet points explaining the video) has been provided that indicates the flows that have been implemented. This allows the teaching staff to mark and is often highly valued in a production company. Please note any areas which are still under development, or not fully implemented. Keep in mind the majority of the paths _should_ be implemented. | 5.0 |
+| Design | The application has a consistent look and feel. You didn't rely on default styles (see below). The flows are not incoherent. This will be a highly subjective area, so feel free to ask for advice before submission (please don't ask the day before, or day of, submission). | 10.0 |
+| | **Total:** | 100.0 | 
+
+### Notes
+
+* If you have `n` parts of the app (e.g. frontend, backend, data pipeline, etc.), regardless of if they are in different repos or just different folders, each section will be worth `worth/n` for each part of the app.
+* If your code is not under the DCSIL organization, it will be considered as non-existent. You will receive a 0 for that part of your codebase.
+* If your application cannot be easily run and evaluated (hosted on an accessible URL, mobile app installable, desktop app installable, etc.), it will be considered incomplete and receive a 0 on all parts that require us to run the software.
+* If your application _is not_ uploading test coverage to codecov.io, it will be considered to have a coverage of 0%
+* The short write up should be adequately long to tell us what to expect and highlight what you think is important. It should be short and concise enough that we're not bored reading it or lose understanding. (Tell a good story). Feel free to include images (and flow charts!!) if you feel they will help.
+* The short write up should be a markdown file in the repository or the body of the release. If it is a markdown file **include it in the release body**
+* Failing to adhere to any of the requirements, even if it's just "record the URL in a certain spot" will result in reasonable penalities. These kind of tedious tasks are in place to help the teaching staff mark, but they're also going to show up sometimes in companies (so get used to them haha). We appreciate your help in making these assignments more streamlined to grade.
```

Comment:
I think it might be good to quantify some of the potential common mistakes we might encounter and their penalties?